src/components/global-header/global-header.stories.mdx

```diff
@@ -0,0 +1,225 @@
+import { useState } from "react";
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import NavigationBar from "../navigation-bar";
+import Box from "../box";
+import VerticalDivider from "../vertical-divider";
+import { Menu, MenuFullscreen, MenuItem, MenuSegmentTitle } from "../menu";
+import GlobalHeader from "./global-header.component";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import LinkTo from "@storybook/addon-links/react";
+import carbonLogo from "../../../logo/carbon-logo.png";
+
+<Meta
+  title="Global Header"
+  parameters={{
+    info: { disable: true },
+    chromatic: { disable: true },
+    docs: {
+      inlineStories: false,
+      iframeHeight: 200,
+    },
+  }}
+/>
+
+# Global Header
+
+Global Header is a wrapper component designed for creating site-wide navigation layouts.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+```javascript
+import GlobalHeader from "carbon-react/lib/components/global-header";
+```
+
+## Examples
+
+### Default
+
+Global Header shares similarities with the <LinkTo kind="Navigation Bar" story="default">Navigation Bar component</LinkTo>, but is permanently fixed to the top of the viewport and has a greater z-index than Navigation Bar.
```

Comment:
```suggestion
Global Header shares similarities with the <LinkTo kind="Navigation Bar" story="default">NavigationBar component</LinkTo>, but is permanently fixed to the top of the viewport and has a greater z-index than NavigationBar.
```

super picky I know but since this is referring to Carbon components I think it should use their actual component name, which of course does not have a space. I actually think the one in the link reads OK with the space and don't mind if that's left, but the sentence at the end I do think should have `NavigationBar` with no space.